src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class Lottery {
+    public static Ball getRandomBall() {
+        String color = ColorSupplier.getRandomColor().toString();
+        int number = new Random().nextInt(100) + 1; // Random number from 1 to 100
+        return new Ball(color, number);
```

Comment:
You don't have to create those temporary variables (`color`, `number`). You should do everything directly in the  `return`.